.ci/test.sh

```diff
@@ -2,6 +2,6 @@
 
 set -e
 
-python3 -c "from PIL import Image"
+python3$1 -c "from PIL import Image"
 
-python3 -bb -m pytest -v -x -W always --cov PIL --cov Tests --cov-report term Tests $REVERSE
+python3$1 -bb -m pytest -v -x -W always --cov PIL --cov Tests --cov-report term Tests $REVERSE
```

Comment:
What's this `$1` for?

Reply:
In [an earlier commit](https://github.com/python-pillow/Pillow/pull/5878/commits/7fd3e9977e7dd0c04e4e72fdc65f59253f0e6281) in this PR, it was used to explicitly call Python with the Python minor version (`python3.8`).
My contribution to the PR removed this.
I think it's only back by accident.

Reply:
An early CI run was failing due to one of the dependencies pulling in python3.9 by accident (so python3 was python3.9, with almost no requirements installed, instead of python3.8, with most dependencies installed.  The dependencies got resolved a while back, so this can go away.